ViAn/Project/video.h

```diff
@@ -32,17 +33,21 @@ struct VideoState {
 
 typedef int ID;
 class Video : Writeable{
```

Comment:
Another completely unrelated question. Do you know the difference between writable and saveable? Right now the saveable have some code for load/save/delete while the writeable is empty. Do we really need both?

Reply:
Saveable inherits from writeable so if we don't want to rewrite saveable we need both